modules/olm-accessing-images-private-registries.adoc

```diff
@@ -18,7 +18,7 @@ Bundle images:: Operator bundle images are metadata and manifests packaged as co
 
 Operator and Operand images:: If an Operator installed from a catalog source uses a private image, either for the Operator image itself or one of the Operand images it watches, the Operator will fail to install because the deployment will not have access to the required registry authentication. Referencing secrets in a catalog source does not enable OLM to place the secrets in target tenant namespaces in which Operands are installed.
 +
-Instead, the authentication details can be added to the global cluster pull secret in the `openshift-config` namespace, which provides access to all namespaces on the cluster. Alternatively, if providing access to the entire cluster is not permissible, the pull secret can be added to the `default` service accounts of the target tenant namespaces.
+Instead, the authentication details can be added to the global cluster pull secret in the `openshift-config` namespace, which provides access to all namespaces on the cluster. Alternatively, if providing access to the entire cluster is not permissible, the pull secret can be added to the `default` service accounts of the target tenant namespaces. //remove?
```

Comment:
In this case, `default` is referring to the service account, so I think this can stay as it is?

Reply:
Oh, I realize you're just spinning on removing this line since you're trying to specify a specific example instead? Do we generally suggest specific namespaces as opposed to `$variableExamples`? Not sure what the common practice is, but if not it may be useful to leave this in.